Signal/src/views/ReminderView.swift

```diff
@@ -73,49 +73,36 @@ class ReminderView: UIView {
         let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleTap(gestureRecognizer:)))
         self.addGestureRecognizer(tapGesture)
 
-        let container = UIView()
+        let container = UIStackView()
+        container.axis = .horizontal
+        container.alignment = .center
+        container.isLayoutMarginsRelativeArrangement = true
 
         self.addSubview(container)
-        container.autoPinWidthToSuperview(withMargin: 16)
-        switch (mode) {
-        case .nag:
-            container.autoPinHeightToSuperview(withMargin: 16)
-        case .explanation:
-            container.autoPinHeightToSuperview(withMargin: 12)
-        }
-
-        // Margin: top and bottom 12 left and right 16.
+        container.layoutMargins = UIEdgeInsets(top: 12, left: 16, bottom: 12, right: 16)
+        container.autoPinToSuperviewEdges()
 
         // Label
         label.font = UIFont.ows_dynamicTypeSubheadline
-        container.addSubview(label)
+        container.addArrangedSubview(label)
         label.textColor = UIColor.black.withAlphaComponent(0.9)
         label.numberOfLines = 0
         label.lineBreakMode = .byWordWrapping
-        label.autoPinLeadingToSuperviewMargin()
-        label.autoPinEdge(toSuperviewEdge: .top)
-        label.autoPinEdge(toSuperviewEdge: .bottom)
```

Comment:
the primary fix is here - we weren't pinning the trailing edge of the label when there wasn't an icon.